modules/fb_instant_articles_display/fb_instant_articles_display.module

```diff
@@ -6,6 +6,49 @@
  */
 
 /**
+* Implements hook_init().
+ *
+ * @todo Update to use a custom DrupalFBInstantArticlesLogger once
+ *   @link https://github.com/facebook/facebook-instant-articles-sdk-php/issues/14 this SDK issue @endlink is in resolved.
+*/
+function fb_instant_articles_display_init() {
```

Comment:
I don't know much about the Logger or what it's useful for.  I need to read up on this area to form an educated opinion about it's implementation.  I think @scottrigby has some opinions about this as well.